corehq/apps/cloudcare/views.py

```diff
@@ -452,44 +452,35 @@ def render_form(request, domain):
 
     session = get_object_or_404(EntrySession, session_id=session_id)
 
-    response = requests.post("{base_url}/webforms/get-xml/{session_id}".format(base_url=get_url_base(),
-                                                                               session_id=session_id))
+    response = requests.post("{base_url}/webforms/get-xml/{session_id}".format(
+        base_url=get_url_base(),
+        session_id=session_id)
+    )
 
     if response.status_code is not 200:
         err = "Session XML could not be found"
         return HttpResponse(err, status=500, content_type="text/plain")
 
-    json_response = json.loads(response.text)
-    xmlns = json_response["xmlns"]
-    form_data_xml = json_response["output"]
+    response_json = json.loads(response.text)
```

Comment:
thought i was going crazy when i tried to do `return json_response(data)` and was getting an error. turned out we were overwriting the `json_response` function haha